src/main/java/org/generationcp/breeding/manager/listmanager/util/GermplasmListExporter.java

```diff
@@ -340,10 +346,34 @@ protected GermplasmList getGermplasmListAndListData(final Integer listId) throws
 
 		final Map<Integer, Variable> variableMap = new HashMap<>();
 		this.addVariableToMap(variableMap, TermId.SEED_AMOUNT_G.getId());
-		this.addVariableToMap(variableMap, TermId.STOCKID.getId());
+		final Variable stockId = createMockStockId();
+		variableMap.put(stockId.getId(), stockId);
 		return variableMap;
 	}
 
+	//	TODO: This method was created to continue export the stockId after removing him FROM cvterm.
+	// 	IBP-4058 Remove StockID variable from the ontology
+	private Variable createMockStockId() {
+		final Variable variable = new Variable();
+		variable.setName(ColumnLabels.STOCKID.getName());
+		variable.setDefinition("ID of an inventory deposit");
```

Comment:
Leaving the note here, as mentioned in the daily, we should analyse in the future if it is perhaps better to still use the cvterm to uniquely identify colums or concepts in bms so that we don't have to create workarounds for them, like negative or null termids, or hardcoded ids in the system (SAMPLES count column, etc)

Reply:
I dont think we should use cvterm for solving a technical problem but I agree we need to find a solution soon because we will have more of this coming